pkg/app/render.go

```diff
@@ -73,7 +74,12 @@ func (a *App) RenderFile(kotsKinds *kotsutil.KotsKinds, inputContent []byte) ([]
 		return nil, errors.Wrap(err, "failed to load appCipher")
 	}
 
-	configCtx, err := builder.NewConfigContext(kotsKinds.Config.Spec.Groups, templateContextValues, localRegistry, appCipher, kotsKinds.License)
+	configGroups := []kotsv1beta1.ConfigGroup{}
+	if kotsKinds.Config != nil {
```

Comment:
I think this should check each level in case people delete the spec in the CR

Reply:
The others aren't `nil`able

Reply:
Actually, yes the groups array was. Checking that now